gatsby-config.js

```diff
@@ -101,7 +101,152 @@ module.exports = {
         // for advanced users.
         //
         // Note: Only the flexsearch engine supports options.
-        engineOptions: "speed",
+        engineOptions: {
+          profile: "score",
+          charset: "latin:advanced",
+          // encode: "extra",
+          tokenize: "reverse",
+          threshold: 11,
+          resolution: 22,
+          depth: 2,
+          filter: [
```

Comment:
I'd expect the library to do this for us if we say we're dealing with Italian. Same thing for the stemmer.

If flexsearch doesn't support this out of the box we should consider upstreaming the support for Italian.

Reply:
Hi @bfabio, the library support `en` and `de` native. We can add our `it` filters by file, or like this, during initialization setup. IMHO, using our Gatsby plugin local-search to integrate flexsearch, and counting that we are using an old and deprecated version 6, this second way, during build time, is the only way right now.